specification/src/main/asciidoc/platform/RelatedDocuments.adoc

```diff
@@ -6,127 +6,129 @@ This specification refers to the following
 
-_Jakarta™ EE Platform Specification Version 8_. Available at: _https://jakarta.ee/specifications/full-platform/8_
+_Jakarta™ EE Web Profile Version 9_. Available at: _https://jakarta.ee/specifications/webprofile/9_
 
 _Java™ Platform, Standard Edition, v8 API Specification (Java SE specification)_. Available at: _https://docs.oracle.com/javase/8/docs/_
 
-_Jakarta™ Enterprise Beans Specification, Version 3.2_. Available at: _https://jakarta.ee/specifications/enterprise-beans/3.2_
+_Java™ Platform, Standard Edition, v11 API Specification (Java SE specification)_. Available at: _https://docs.oracle.com/en/java/javase/11/_
 
-_Jakarta™ Server Pages Specification, Version 2.3_. Available at: _https://jakarta.ee/specifications/pages/2.3_
+_Jakarta™ Enterprise Beans Specification, Version 4.0_. Available at: _https://jakarta.ee/specifications/enterprise-beans/4.0_
 
-_Jakarta™ Expression Language Specification, Version 3.0_. Available at: _https://jakarta.ee/specifications/expression-language/3.0_
+_Jakarta™ Server Pages Specification, Version 3.0_. Available at: _https://jakarta.ee/specifications/pages/3.0_
 
-_Jakarta™ Annotations Specification, Version 1.3_. Available at: _https://jakarta.ee/specifications/annotations/1.3_
+_Jakarta™ Expression Language Specification, Version 4.0_. Available at: _https://jakarta.ee/specifications/expression-language/4.0_
 
-_Jakarta™ Servlet Specification, Version 4.0_. Available at: _https://jakarta.ee/specifications/servlet/4.0_
+_Jakarta™ Annotations Specification, Version 2.0_. Available at: _https://jakarta.ee/specifications/annotations/2.0_
+
+_Jakarta™ Servlet Specification, Version 5.0_. Available at: _https://jakarta.ee/specifications/servlet/5.0_
 
 _JDBC™ 4.2 API (JDBC specification)_. Available at: _https://jcp.org/en/jsr/detail?id=221_
 
 _Java™ Naming and Directory Interface 1.2 Specification (JNDI specification)_. Available at: _https://docs.oracle.com/javase/8/docs/technotes/guides/jndi/index.html_
 
-_Jakarta™ Messaging Specification, Version 2.0_. Available at: _https://jakarta.ee/specifications/messaging/2.0_
+_Jakarta™ Messaging Specification, Version 3.0_. Available at: _https://jakarta.ee/specifications/messaging/3.0_
 
-_Jakarta™ Transaction Specification, Version 1.3_. Available at: _https://jakarta.ee/specifications/transactions/1.3_
+_Jakarta™ Transaction Specification, Version 2.0_. Available at: _https://jakarta.ee/specifications/transactions/2.0_
 
 _Java™ Transaction Service, Version 1.0_ (JTS specification)_. Available at: _https://www.oracle.com/technetwork/java/javaee/jts-spec095-1508547.pdf_
 
-_Jakarta™ Mail Specification, Version 1.6_. Available at: _https://jakarta.ee/specifications/mail/1.6_
+_Jakarta™ Mail Specification, Version 2.0_. Available at: _https://jakarta.ee/specifications/mail/2.0_
 
 _JavaBeans™ Activation Framework Specification Version 1.2 (JAF specification)_. Available at: _https://www.oracle.com/technetwork/java/javase/tech/index-jsp-138795.html_
```

Comment:
This should be updated to our Jakarta Activation 2.0.  We know the eventual url, even if it's not integrated/merged yet...

Reply:
yes, missed that one